src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
```

Comment:
There are quite a few lines with whitespace changes here, but I wasn't sure if you had any other changes in the file?  Can it be removed from our PR?

Reply:
Yeah these changes happened automatically when I merged with main I believe? No other changes were made to this file.

Reply:
These occurred automatically when merging with main, all the contents of the file are the same. Some lines got delete and reinserted back in the same spot but still counted as changes for some reason.